server/routes/matches.ts

```diff
@@ -0,0 +1,9 @@
+import express, { Router } from 'express';
+import { errorWrapper } from '../utils';
+import { getStadiumMatches } from '../controllers';
+
+const matchRouter: Router = express.Router();
+
+matchRouter.get('/stadium/:stadiumId', errorWrapper(getStadiumMatches));
```

Comment:
`.get('/stadium/matches/:stadiumId', errorWrapper(getStadiumMatches));`
it will better and more clean when follow this naming convention.
and better to include this route in the stadiums router